src/getHumanAge.test.js

```diff
@@ -3,8 +3,27 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('Should return [0, 0] if age < 15', () => {
```

Comment:
Imagine that your test fails with such text "should return [0, 0] if age < 15" how to understand exact age? Is it 14 or 10? try to use exact values and better to create constants to reuse them in the test name and expect() function

Reply:
Fixed